projects/storefrontapp/src/app/app.module.ts

```diff
@@ -57,6 +56,7 @@ if (!environment.production) {
     ...devImports,
   ],
   providers: [
+    provideHttpClient(withFetch()),
```

Comment:
During manual QA, I discovered that I cannot complete an user registration process on the Register route.
I get CORS error.

I checked that:
- I get this error, when I have `provideHttpClient(withFetch()),` or `provideHttpClient(),`
- I don't get this error, when I have `HttpClientModule` instead

I checked that the error is gone, if I add `withInterceptorsFromDi()`.

Reply:
Good catch. That's the reason why e2e tests were failing.
Apart from that, when adding the `withInterceptorsFromDi()` config, I noticed this in its description:
```
Prefer `withInterceptors` and functional interceptors instead, as support for DI-provided interceptors may be phased out in a later release.
```
We should remember this in the future.